third_party/blink/renderer/core/resource_pool_limiter/resource_pool_limiter.h

```diff
@@ -51,7 +51,7 @@ class CORE_EXPORT ResourcePoolLimiter {
 
   void DropResourceInUse(const ResourceInUseTracker* resource_in_use_tracker);
 
-  Mutex resources_in_use_lock_;
+  base::Lock resources_in_use_lock_;
   HashMap<String, int> resources_in_use_;
```

Comment:
please make it `HashMap<String, int> resources_in_use_ GUARDED_BY(resources_in_use_lock_);`

Reply:
Fixed in e0a4be0e17